### PR TITLE
Mabatistussi/blink fade sketches

### PR DESCRIPTION
**Pull request by mabatistussi**

### CODE REVIEW

File: fade.ino [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  // change the brightness for next time through the loop:
+  brightness = brightness + fadeAmount;
+
+  // reverse the direction of the fading at the ends of the fade:
+  if (brightness <= 0 || brightness >= 255) {
+    fadeAmount = -fadeAmount;
+  }
+  // wait for 30 milliseconds to see the dimming effect
+  delay(30);
+}
```
Comment (CrisGavril) [inline fade.ino:25]: Hey Marisa, your code looks great. You seem more confident than me with the Java syntax :)

File: fade.ino [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  analogWrite(led, brightness);
+
+  // change the brightness for next time through the loop:
+  brightness = brightness + fadeAmount;
+
+  // reverse the direction of the fading at the ends of the fade:
+  if (brightness <= 0 || brightness >= 255) {
+    fadeAmount = -fadeAmount;
+  }
+  // wait for 30 milliseconds to see the dimming effect
```
Comment (kimberleynyam) [inline fade.ino:23]: Love the comments, they're very clear!

File: fade.ino [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+// the setup routine runs once when you press reset:
+void setup() {
+  // declare pin 9 to be an output:
+  pinMode(led, OUTPUT);
+}
+
+// the loop routine runs over and over again forever:
+void loop() {
+  // set the brightness of pin 9:
+  analogWrite(led, brightness);
```
Comment (Levon2014) [inline fade.ino:14]: Everything is good and tiding so keep up the good work.

**Comment by mabatistussi** [comment]
Thanks guys for your comments!!Really helpful

**Comment by LeticiaBriscoe** [comment]
Doing well mariss

File: fade.ino (hunk 1 above)
Comment (Johntwity) [inline fade.ino:25]: Great coding Marisa you have done well keep up the good work Marisa.

Comment (Oye247) [inline fade.ino:25]: Nice Work Marisa! Your work is easy to follow.

**Comment by aliblackwell** [comment]
Well done @mabatistussi! This is excellent.

**Pull request merged**